flink-runtime/src/test/java/org/apache/flink/runtime/state/FileStateBackendMigrationTest.java

```diff
@@ -18,18 +18,24 @@
 package org.apache.flink.runtime.state;
 
 import org.apache.flink.runtime.state.filesystem.FsStateBackend;
+import org.apache.flink.testutils.junit.extensions.parameterized.Parameters;
 
-import java.io.File;
+import java.util.Arrays;
+import java.util.Collection;
 
 /**
  * Tests for the keyed state backend and operator state backend, as created by the {@link
  * FsStateBackend}.
  */
 public class FileStateBackendMigrationTest extends StateBackendMigrationTestBase<FsStateBackend> {
 
+    @Parameters
+    public static Collection<Object> modes() {
+        return Arrays.asList("just to please junit5 parameter test in base class");
```

Comment:
I don't think this sentence could be understood well.

Reply:
I have changed the expression in the parameter list to enhanace reabability.

```
    @Parameters
    public static Collection<Object> modes() {
        // current test does not need to be parameterized
        return Collections.singletonList("no-use");
    }
```